static/app/utils/useTeams.tsx

```diff
@@ -74,10 +76,14 @@ type Options = {
   /**
    * When true, fetches user's teams if necessary and only provides user's
    * teams (isMember = true).
+   *
+   * @deprecated use `useUserTeams()`
    */
   provideUserTeams?: boolean;
   /**
    * When provided, fetches specified teams by slug if necessary and only provides those teams.
+   *
+   * @deprecated use `useTeamsById({slugs: []})`
```

Comment:
With `with `teamDetails.tsx` being updated below, is anything else still using this?

Reply:
there's two places left using it via the old `<Teams ids=` and `<Teams slugs=`